uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/LeavePeriodEntitlementTest.php

```diff
@@ -755,14 +782,36 @@ public function testgetLeavePeriodEntitlementRemainder() {
     );
 
     $params = ['entitlement_id' => $periodEntitlement->id];
-    $result = $periodEntitlement::getLeavePeriodEntitlementRemainder($params);
+    $result = $periodEntitlement::getRemainder($params);
```

Comment:
You're still calling the static method from an instance here. Please make sure to check other places where this is happening and fix it